cmd/api-server/services/controlplane.go

```diff
@@ -293,10 +293,10 @@ func CreateControlPlane(ctx context.Context, cfg *config.Config, features featur
 			return
 		}),
 		cloudtestworkflow.CmdTestWorkflowExecutionInsert: controlplane.Handler(func(ctx context.Context, data cloudtestworkflow.ExecutionInsertRequest) (r cloudtestworkflow.ExecutionInsertResponse, err error) {
-			return r, testWorkflowResultsRepository.Insert(ctx, data.WorkflowExecution)
+			return r, testWorkflowResultsRepository.Insert(ctx, *data.WorkflowExecution.Clone())
```

Comment:
Is there a reason to clone it there? It's serialized immediately.

Reply:
will change